libledger/DBInitializer.cpp

```diff
@@ -264,6 +264,7 @@ void DBInitializer::initZdbStorage()
         m_param->mutableStorageParam().maxConnections};
 
     auto sqlconnpool = std::make_shared<SQLConnectionPool>();
+    sqlconnpool->createDataBase(zdbConfig);

```

Comment:
what about make createDataBase() call in InitConnectionPool(), in case someone modify code don't know to call this  createDataBase()